js/SliderModel.js

```diff
@@ -153,6 +153,24 @@ export default class SliderModel extends QuestionModel {
     this.set('_score', score);
   }
 
+  getInteractionObject() {
+    return {
+      correctResponsesPattern: this.getCorrectResponsesPattern()
+    }
+  }
+
+  getCorrectResponsesPattern() {
+    const correctAnswer = this.get('_correctAnswer');
+    if (correctAnswer) return [correctAnswer];
+    const correctRange = this.get('_correctRange');
+    if (!correctRange) return null;
+    const bottom = correctRange?._bottom ?? '';
+    const top = correctRange?._top ?? '';
```

Comment:
The `?.` isn't necessary as both `null` and `undefined` are falsies, which would return on line 166.